.eslintrc

```diff
@@ -0,0 +1,33 @@
+---
+
+root: true
+
+env:
+  browser: true
+  mocha: true
+  node: true
+
+rules:
+  comma-dangle: 0
```

Comment:
This one should go, but I don't want to add extra delay to your PR. Could you add the comment `# TODO Fix trailing commas and remove this line` at the end of this line please?